src/viewers/VivViewer.js

```diff
@@ -193,7 +183,7 @@ export default class VivViewer extends PureComponent {
       deckGLViews[0] = deckGLViews[randomizedIndex];
       deckGLViews[randomizedIndex] = holdFirstElement;
     }
-    return !isSafari() ? (
+    return (
       <DeckGL
         glOptions={{ webgl2: true }}
```

Comment:
I guess related to above, is there a way to check the `glOptions` in our layers, and then do the feature detection here?